Tests/SwiftCliCoreTests/NestedArrayBuilderTest.swift

```diff
@@ -24,29 +24,8 @@ class MultipleStitchExpanderTest: XCTestCase {
         expect(result).to(equal(["k1", "k1", "k1", "k1", "p1"]))
     }
 
-    func testInvalidZeroCountStitch() throws {
-        expect{
-            try NestedArrayBuilder().expandRow(row: ["k0", "p1"])
-        }
-        .to(throwError(
-            InputError.invalidStitchNumber(
-                rowNumber: nil,
-                invalidStitch: "k0",
-                validStitchType: "k",
-                invalidStitchNumber: "0")
-        ))
-    }
-
-    func testInvalidWordCountStitch() throws {
-        expect{
-            try NestedArrayBuilder().expandRow(row: ["kx", "py"])
-        }
-        .to(throwError(
-            InputError.invalidStitchNumber(
-                    rowNumber: nil,
-                    invalidStitch: "kx",
-                    validStitchType: "k",
-                    invalidStitchNumber: "x")
-        ))
+    func testExpandInvalidMultipleStitch() throws {
+        let result = try NestedArrayBuilder().expandRow(row: ["k0", "p1"])
+        expect(result).to(equal(["k0", "p1"]))
```

Comment:
A little unclear on what's going on here. Shouldn't this be throwing an error, not just returning?

Reply:
This test is misleading at this point. NestedArrayBuilder used to throw errors, but it no longer does that - it simply leaves incorrect stitches to be caught after I run the next function in `InputValidator.validateInput`. Should I remove this test because it is confusing? Or is it better to throw errors within `NestedArrayBuilder` again?  I wonder what makes the most sense?

Reply:
I'm going to remove this test for now because `NestedArrayBuilder` is no longer responsible for finding errors and it's confusing.